### DISCUSSION TITLE
Not integrating in Calendar App in Pixel 9 Pro

### DISCUSSION THREAD

**Discussion by mrHJS** [bitfireAT/icsx5#415] [General]
I installed ICSx5 on my new Pixel 9 Pro and added a Calendar. However, in the native Calendar App the subscribed Calendar will not show up. I already synced various times. Is there anything I missed.

Thank you

**Comment by mrHJS** [comment]
Oh man, forget it... Had to activate Cal subscriptions... Silly me

**Reply by sunkup** [reply]
Good to hear you found out :)
